packages/wcg-crypto/index.android.ts

```diff
@@ -0,0 +1,27 @@
+declare const org;
```

Comment:
Worth adding whichever lib provides the `Crypto` type. Not sure which lib it is, so you can see all available libs [here](https://www.typescriptlang.org/tsconfig#lib).
```suggestion
/// <reference lib="dom" />
declare const org;
```